fuzzers/fuzzbench/src/lib.rs

```diff
@@ -351,17 +354,18 @@ fn fuzz(
     // The order of the stages matter!
     let mut stages = tuple_list!(calibration, tracing, i2s, power);
 
-    // Autodict Tokens
-    let autodict_tokens = unsafe { Tokens::from_autodict(DICT_START, DICT_STOP)? };
-
     // Read tokens
     if let Some(tokenfile) = tokenfile {
         if state.metadata().get::<Tokens>().is_none() {
             state.add_metadata(Tokens::from_tokens_file(tokenfile)?);
         }
     }
 
-    state.add_metadata(autodict_tokens);
+    // Autodict Tokens
+    if cfg!(target_os = "linux") {
+        let autodict_tokens = unsafe { Tokens::from_autodict(DICT_START, DICT_STOP)? };
```

Comment:
Shouldn't this fn take no parameters ( and be called just `Tokens::auto_tokens`)?
The range is an implementation detail, and then it doesn't need to be `unsafe`, either

Reply:
Maybe just have a `fn auto_tokens` in targets that returns them